postgres-appliance/Dockerfile

```diff
@@ -96,7 +97,9 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         && git clone -b $DECODERBUFS https://github.com/debezium/postgres-decoderbufs.git \
         && git clone -b $PLPROFILER https://github.com/bigsql/plprofiler.git \
         && git clone -b $PLPGSQL_CHECK https://github.com/okbob/plpgsql_check.git \
-        && git clone git://www.sigaev.ru/plantuner.git \
+        && mkdir plantuner \
+            && curl -sL https://github.com/gertvdijk/plantuner/archive/$PLANTUNER_COMMIT.tar.gz \
```

Comment:
I would better prefer keeping the tarball in the `postgres-appliance/dependencies/`

Reply:
Okay, I will change that. :) Thank for your review.